packages/vue/package.json

```diff
@@ -62,7 +62,7 @@
     "@vueuse/core": "^8.4.2",
     "@vueuse/rxjs": "^8.2.0",
     "@web3-onboard/common": "^2.3.3",
-    "@web3-onboard/core": "^2.21.2",
+    "@web3-onboard/core": "^2.21.3-alpha.1",
```

Comment:
Can you please bump the package version following the alpha release flag up on line 3 of this file?